apps/backend-docker/src/app.ts

```diff
@@ -57,6 +57,10 @@ function prepareApp({ prisma, redisExec, pubSub, cache, emitter }: any) {
           // }
 
           if (req.cookies) {
+            if (req.cookies['no-account']) {
```

Comment:
if the account has been deleted, we set this cookie to ensure that no account is immediately recreated in, e.g., the OLAT environment (where we try to create an account on each page load). does not cover the case where you visit using a different browser or after cleaning cookies.

Reply:
removed because of change in account creation approach with dedicated page